README.md

```diff
@@ -244,10 +244,9 @@ to create a free account to add new tickets.
 
 ## Releasing
 
-1. Make sure version, changelog, etc. have been updated.
-1. Commit and tag with new version number: e.g. `v1.2.3`
-1. Push tag to Github: `git push upstream --tags` (where `upstream` is the remote name of the puppetlabs fork of this repo)
-1. Wait for Travis CI to test and push new release to Rubygems.
+1. Run the release_prep GitHub Action with the new version number, for example, 1.2.3. It will create a new pull request
```

Comment:
can we also put link here, so that user doesn't need to find the action as well?

Also we should put note around how next version gets calculated?

Reply:
For next version we follow https://semver.org/ and it is mentioned in the changelog yes link for action can be added

Reply:
Added the links